README.md

```diff
@@ -870,18 +928,39 @@ Branch.getInstance().getReferralCodeWithPrefix("BRANCH",
 })
 ```
 
-### Validate referral code
+####Parameters 
+
+You can also tune the referral code to the finest granularity, with the following additional parameters:
+
+**bucket** _NSString*_
+: The name of the bucket to use. If none is specified, defaults to 'default.'
+
+**calculation_type**  _ReferralCodeCalculation_
+: This defines whether the referral code can be applied indefinitely, or only once per user.
+
+1. _BranchUnlimitedRewards_ - referral code can be applied continually.
+1. _BranchUniqueRewards_ - a user can only apply a specific referral code once.
+
+**location** _ReferralCodeLocation_
+: The user to reward for applying the referral code.
+
+1. _BranchReferreeUser_ - the user applying the referral code receives credit.
+1. _BranchReferringUser_ - the user who created the referral code receives credit.
+1. _BranchBothUsers_ - both the creator and applicant receive credit.
+
+
+### Validate Referral Code
 
 Validate if a referral code exists in Branch system and is still valid.
 A code is vaild if:
 
-1. It hasn't expired.
-1. If its calculation type is uniqe, it hasn't been applied by current user.
+* It hasn't expired.
+* If its calculation type is uniqe, it hasn't been applied by current user.
```

Comment:
Typo: unique.

Also, not sure if this is important, but it may be useful to mention it's also valid if it's type unlimited, as long as it hasn't expired.